features/facebookImport/test/utils/analyses-execution.js

```diff
@@ -41,11 +41,11 @@ export async function runAnalysisForAccount(
 }
 
 export async function runAnalysesForZip(zipFile) {
-    const facebookAccount = await importZip({
-        dataImporters,
+    const facebookAccount = new FacebookAccount();
+    await facebookAccount.import({
```

Comment:
If I have understood this, this should return a result, right? So you're not actually doing anything here that affects the rest of the variables?

Reply:
Well, I was rather asking if facebookAccount was mutated by calling *import*. I though `import` had changed to return a result...

Reply:
Yeah different import thing :D